lib/Configuration.ts

```diff
@@ -103,11 +103,11 @@ export class Configuration {
         // tslint:disable-next-line:no-http-string
         return "http://api.local.dev:3000";
       case "sandbox":
-        return "https://api.railz.io";
+        return "https://api.trolley.com";
```

Comment:
@Sheikh-Aman I also updated this one -- Does this look good?

If so, this is ready to merge into dev and I'm good to start the scoped JS work 🥳

Reply:
Yeah looks good.
This will anyway be deleted when we remove environments from SDKs